index.html

```diff
@@ -1,21 +1,66 @@
-<html>
-<head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
-</head>
-<body>
-<div class="container">
-    <header></header>
-    <section class="section-1">
-        <h1>I'm Marine Glad to see you</h1>
-        <p>I'm a software Developer! I can help you build aproduct, 
-            feature or website look through some of my work and 
-            exprience! if you like what you se & have aproject you need, 
-            don't hestiate to contact me</p>
-    </section>
-
-</div>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;1,300&display=swap"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <div class="logo">
+            <img
+              src="Asset/My Logo.png"
+              alt="log for My portifolio"
+              class="logo"
+            />
+          </div>
+          <span class="menu-icon">≡</span>
+        </nav>
+        <section class="Content">
+          <h2>I'm Melkamu Alemawu Ayen!</h2>
```

Comment:
- [ ] I would kindly suggest you make the main title an `h1` tag. 
     - The main title (e.g., "I'm Marianne ...") is a `h1` tag. * If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

Reply:
yes i am using h1 tag